docs/style.css

```diff
@@ -1,1086 +1,1638 @@
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  padding-left: 8px;
+  padding-right: 16px;
+  background-color: var(--light);
+  color: var(--dark);
+  min-width: 108px;
+  max-width: 141px;
+  height: 46px;
+  border: 1px solid #c1b6ad;
+  border-radius: 100px;
+}
+
+.modal-size-label.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  padding-left: 8px;
+  padding-right: 16px;
+  background-color: var(--light);
+  color: var(--dark);
+  min-width: 108px;
+  max-width: 141px;
+  height: 46px;
+  border: 1px solid #c1b6ad;
+  border-radius: 100px;
+}
+
+.modal-size-label.button:hover {
+  background-color: var(--container);
+  color: var(--light);
+}
+
+.coffee-size {
+  background-color: #c1b6ad;
+  padding: 2px 9px;
+  border-radius: 100px;
+}
+
+.modal-size-label.button:hover .coffee-size {
+  background: #e1d4c9;
+  color: var(--dark);
+}
+
+label.modal-size-label + input[type="radio"]:checked {
+  background: var(--container);
+  color: var(--light);
+}
+
+input[type="radio"]:checked + .coffee-size {
+  background: #e1d4c9;
+  color: var(--dark);
+}
+
+.modal-size-btn:focus {
+  background-color: var(--container);
+  color: var(--light);
+}
+
+.modal-size-btn:focus .coffee-size {
+  background: #e1d4c9;
+  color: var(--dark);
+}
+
+.modal-size-btn:hover .coffee-size {
+  background: #e1d4c9;
+  color: var(--dark);
+}
+
+.modal-size-btn:active .coffee-size {
+  background: #e1d4c9;
+  color: var(--dark);
+}
+
+.modal-Additives-input,
+.modal-size-btn.button {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+}
+
+.modal-Additives-label.button:hover {
+  background-color: var(--container);
+  color: var(--light);
+}
+
+.modal-Additives-checked {
+  background-color: var(--container);
+  color: var(--light);
+}
+
+.coffee-Additives.button {
+  background-color: #c1b6ad;
+  padding: 2px 9px;
+  border-radius: 100px;
+}
+
+.modal-Additives-label.button:focus {
+  background-color: var(--container);
+  color: var(--light);
+}
+
+.modal-Additives-label.button:hover .coffee-Additives.button {
+  background-color: #e1d4c9;
+  color: var(--dark);
+}
+
+input[type="checkbox"]:checked + .coffee-Additives.button {
+  background-color: #e1d4c9;
+  color: var(--dark);
+}
+
+.modal-Additives-label input[type="checkbox"]:checked {
+  background-color: #665f55;
+  color: var(--light);
+}
+
+.total {
+  display: flex;
+  justify-content: space-between;
+}
+
+.alert {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  padding-top: 0;
+  padding-bottom: 12px;
+}
+
+.alert-p {
+  margin-top: 0;
+  padding: 0;
+  text-align: justify;
+}
+
+.modal-hr {
+  border-top: 1px solid #c1b6ad;
+  margin: 0;
+}
+
+.modal-Close {
+  background: var(--light);
+  border: 1px solid #c1b6ad;
+  border-radius: 100px;
+  margin-top: 12px;
+  height: 44px;
+}
+
+.modal-Close:hover {
+  color: var(--dark);
+}
+
+.Additives-p {
+  margin-bottom: 10px;
+}
+
+.Additives-button-set {
```

Comment:
class naming convention. Please review the file to find inconsistencies

Reply:
There are still a mix of kebab-case and camelCase naming, but lets leave it. Please make sure to stick to one convention for the future.